site/website/preview-landing/index.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (C) 2007-2019 Crafter Software Corporation. All Rights Reserved.
```

Comment:
Fix this copyright notice with the correct year and language.

Reply:
Done.